lambda/MAS.Tests/UnitTests/S3ServiceTests.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Amazon.S3;
+using Amazon.S3.Model;
+using MAS.Services;
+using MAS.Models;
+using MAS.Tests.Infrastructure;
+using Moq;
+using System.Net;
+using System.Threading;
+using Xunit;
+using Shouldly;
+
+namespace MAS.Tests.UnitTests
+{
+    public class S3ServiceTests : TestBase
+    {
+        [Fact]
+        public void WriteItemToS3()
+        {
```

Comment:
Where's the SUT here? It seems we're testing the amazon Client here which is unnecessary.

Reply:
I've removed the test as like you said we are only testing 3rd party here